File: packages/gunreip/laravel-translation-workbench/resources/views/pages/tw-graph/samples/documentation/idea-to-paper/flow/switch-case/code-examples/switch-grouped-3.cpp
enum class Status { Draft, Review, Revision, Published, Unknown };
Status status = Status::Draft;

switch (status) {
    case Status::Draft:
    case Status::Review:
    case Status::Revision:
        openEditor();
        break;
    case Status::Published:
        displayArticle();
        break;
    default:
        showStatusHint();
        break;
}

// Continue after the selected action sequence.
continueProcess();

File: packages/gunreip/laravel-translation-workbench/resources/views/pages/tw-graph/samples/documentation/idea-to-paper/flow/switch-case/code-examples/switch-grouped-3.cs
string status = "draft";

switch (status) {
    case "draft":
    case "review":
    case "revision":
        OpenEditor();
        break;
    case "published":
        DisplayArticle();
        break;
    default:
        ShowStatusHint();
        break;
}

// Continue after the selected action sequence.
ContinueProcess();
